### ISSUE TITLE
Feature Request: also show original string after translating it

### ISSUE THREAD

**Issue by m-rey** [sunilpaulmathew/Translator#23]
Often times it is useful to also be able to see the original string after it was translated.

At the moment, after a translation string is translated and the list of translation strings is redrawn (for example by entering and leaving the options overflow menu), there's no easy way to see what see what the original string was or to recheck whether the entered translation fits the given string.

A way to tackle this request could be by showing the original string above the textbox used for setting the translation string, instead of just prefilling the textbox with the original string.

**Comment by sunilpaulmathew** [comment]
@m-rey 
Thanks for this suggestion which I already received from others (by email) as well.
At this moment, for the sake of simplicity, the app will simply replace the original word from the original text once the user clicks the "Update" button. Of course, it is definitely possible,  but only after making some core changes in the app's current design. I will consider it, but only after dealing with some other issues that need to be addressed immediately to satisfy Google (latest policy changes) to ensure the presence of the app in Googler Play. We will see in future.

**Comment by franco27** [comment]
Yes, that's true, it would be useful. It would be good to have a button to copy the text to the clipboard as well.
